python/pyspark/ml/classification.py

```diff
@@ -16,19 +16,24 @@
 #
 
 import operator
+import sys
+import uuid
```

Comment:
@Fokko, do you mind opening a minor PR to add this `uuid` to all other branches - looks like it wasn't there from the very first place https://github.com/Fokko/spark/commit/90b46e014a60069bd18754b02fce056d8f4d1b3e? 

I checked other occurrences fixed here, and other ones look fine to not fix in other branches.

Reply:
Sure, I've created two PR's:

- 3.0 branch: https://github.com/apache/spark/pull/29615
- 2.4 branch: https://github.com/apache/spark/pull/29616

Reply:
Oh @Fokko sorry I misread. It was fine because we're using wildcard imports at `from pyspark.ml.util import *` and `util` imports `uuid`. So it didn't crash before ..  Let's just don't port this back alone :-)..